form/tailwindcss2.rst

```diff
@@ -0,0 +1,95 @@
+Tailwind CSS 2 Form Theme
+=========================
+
+.. versionadded:: 5.3
+
+    The Tailwind CSS 2 Form Theme was introduced in Symfony 5.3 as an
+    :doc:`experimental feature </contributing/code/experimental>`.
```

Comment:
Is this true? IIRC we cannot add experimental features in the last minor before an LTS release 🤔

Reply:
Only LTS is not allowed to be experimental

Reply:
If I would add sth experimental means I can break BC in the next minor, which would be the next LTS which is not possible as far as I know. Plus we should keep experimental only for one minor.

I would then break the BC in an LTS release.

I understand it like experimental can only be introduced in 5.0, 5.1 and 5.2 but I can be wrong

Reply:
It's perfectly fine to break BC going into an LTS (if it's experimental). The only reason experimental is not allowed in LTS releases is because these must be the absolute top of stability in the Symfony releases. That doesn't quite fit in with having "experimental" stuff in there.

(however, not all features need to be introduced as experimental)